README.md

```diff
@@ -1,28 +1,30 @@
-# opbeat.js
+# OpbeatJS
 
 [![Build status](https://travis-ci.org/opbeat/opbeat-js.svg?branch=master)](https://travis-ci.org/opbeat/opbeat-js)
 [![js-standard-style](https://img.shields.io/badge/code%20style-standard-brightgreen.svg?style=flat)](https://github.com/feross/standard)
 
-opbeat.js is the official JavaScript agent for Opbeat. This agent enables automatic exception collection and framework-specific code instrumentation of your front-end code.
+OpbeatJS is the official JavaScript agent for Opbeat. This agent enables framework-specific code instrumentation of your front-end code and automatic exception collection.
```

Comment:
`OpbeatJS is the official JavaScript module for Opbeat. This module enables framework-specific performance metrics of your client-side application and automatic exception collection.`